dist/t/spec/features/0040_package_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "spec_helper"
+
+RSpec.describe "Package" do
+
+  scenario "should be able to create new" do
+    login('Admin')
+    within("div#subheader") do
+      click_link('Home Project')
+    end
+    expect(current_path).to eq '/project/show/home:Admin'
+    click_link('Create package')
+    fill_in 'name', with: 'ctris'
+    fill_in 'title', with: 'ctris'
+    fill_in 'description', with: 'ctris'
+    click_button('Save changes')
+    expect(page).to have_content("Package 'ctris' was created successfully")
+
+    logout
+  end
+
+  scenario "should be able to upload files" do
+    login('Admin')
+    within("div#subheader") do
+      click_link('Home Project')
+    end
+    expect(current_path).to eq '/project/show/home:Admin'
+    click_link('ctris')
+    expect(current_path).to eq '/package/show/home:Admin/ctris'
```

Comment:
If this test is really just about uploading we could directly go to that page instead. Or, if we think it's worth to test this, move it into a shared context.

That said, I am ok to do this in a separate PR.